http-core/src/main/scala/org/apache/pekko/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -122,12 +122,18 @@ private[http] object Http2Blueprint {
       telemetry: TelemetrySpi,
     dateHeaderRendering: DateHeaderRendering): BidiFlow[HttpResponse, ByteString, ByteString, HttpRequest, ServerTerminator] = {
     val masterHttpHeaderParser = HttpHeaderParser(settings.parserSettings, log) // FIXME: reuse for framing
-    telemetry.serverConnection atop
+    val flow0 = telemetry.serverConnection atop
       httpLayer(settings, log, dateHeaderRendering) atopKeepRight
       serverDemux(settings.http2Settings, initialDemuxerSettings, upgraded) atop
       FrameLogger.logFramesIfEnabled(settings.http2Settings.logFrames) atop // enable for debugging
-      hpackCoding(masterHttpHeaderParser, settings.parserSettings) atop
-      framing(log) atop
+      hpackCoding(masterHttpHeaderParser, settings.parserSettings)
+
+    val frameTypesForThrottle = getFrameTypesForThrottle(settings.http2Settings)
+    val flow1 = if (frameTypesForThrottle.nonEmpty) {
+      flow0 atop rapidResetMitigation(settings.http2Settings, frameTypesForThrottle) atopKeepLeft framing(log)
+    } else flow0 atop framing(log)
+
+    flow1 atop
```

Comment:
Can `flow0` and `flow1` be replaced with a more meaningful name?

Reply:
is cdfca2d098eacae695fb4c09cc37276e2d4f2d2c ok?